examples/send_op_return/send_op_return.go

```diff
@@ -20,7 +20,11 @@ func main() {
 
 	const server = "http://localhost:3003/v1"
```

Comment:
Maybe let's not make it a part of this PR, but I think it might be easier to extract the server url to some other file and share it in examples? Right now its `http://localhost:3003/v1` but as we discussed today it's actually `http://localhost:3003/api/v1` so it would be easier to change. It's a minor and just an idea. Let me know WDYT

Reply:
Yes, I confirm - it's a part of SPV-904. Damian asked that go-client should "know" which sufix it should use. Currently we support both: v1 (AdminAPI) and /api/v1 (userAPI).

Reply:
Ok - so I'm resolving it this comment 👍